econml/_cate_estimator.py

```diff
@@ -938,9 +938,12 @@ def summary(self, alpha=0.1, value=0, decimals=3, feature_names=None, treatment_
                                                               treatment_names=treatment_names,
                                                               output_names=output_names)
             coef_array = coef_table.values
-            coef_headers = [i + '\n' +
-                            j for (i, j) in coef_table.columns] if d_t > 1 else coef_table.columns.tolist()
-            coef_stubs = [i + ' | ' + j for (i, j) in coef_table.index] if d_y > 1 else coef_table.index.tolist()
+            coef_headers = coef_table.columns.tolist()
+            n_level = coef_table.index.nlevels
+            if n_level > 1:
+                coef_stubs = ["|".join(i) for i in coef_table.index.values]
```

Comment:
[minor] I think `i` is kind of a weird name for this variable; usually I'd expect a variable called `i` to be a number or some other simple item, but it looks like here it's a pair - is there a more descriptive name you could use to make it more clear what's happening here?

Reply:
done